lib/execution_engine2/db/models/models.py

```diff
@@ -109,7 +109,6 @@ class Meta(EmbeddedDocument):
     token_id = StringField()
     tag = StringField()
     cell_id = StringField()
-    status = StringField()
```

Comment:
No CI jobs have this field and per @briehl the Narrative doesn't use it, so removed